VALIDATORS_FAQ.md

```diff
@@ -110,35 +119,43 @@ Even though delegated funds cannot be stolen by their validators, delegators are
 
 The validator that is selected to propose the next block is called proposer. Each proposer is selected deterministically, and the frequency of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 Atoms and a validator's total stake is 10 Atoms, then this validator will be chosen 10% of the time as the next proposer.
 
+### Will validators of the Cosmos Hub ever be required to validate other zones in the Cosmos ecosystem?
+
+Yes, they will. If governance decides so, validators of the Cosmos hub might have to validate other zones in the Cosmos ecosystem. For example, validators of the Cosmos hub will also validate the first public Ethermint zone. Of course, they will be compensated for it in the form of block rewards and transaction fees. These rewards will be paid in the native currency of the Ethermint zone, called Photons.
+
+
+## Staking and incentives
+
 ### What is the incentive to stake?
 
-Each member of a validator’s staking pool earns different types of rewards:
-* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+Each member of a validator’s staking pool earns different types of revenue:
+* **Block provisions:** Native tokens of applications run by validators (e.g. Atoms on the Cosmos Hub) are inflated to produce block provisions. These provisions exist to incentivize Atom holders to bond their stake, as non-bonded Atom will be diluted over time. 
+* **Block rewards:** Block rewards are paid in Photons. Initial distribution of Photons will be hard spooned from Ethereum. This means Photons will be emitted 1:1 to Ether. 
 * **Transaction fees:** The Cosmos Hub maintains a whitelist of token that are accepted as fee payment. 
 
-These rewards are divided among validators' staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake. Note that a commission on fee rewards for delegators is applied by the validator before they are distributed. However, no commission can be applied on block provisions (see below).
+This total revenue is divided among validators' staking pools according to each validator’s weight. Then, it is divided among delegators in proportion to each delegator’s stake. Note that a commission on delegators' revenue is applied by the validator before it is distributed. 
```

Comment:
Then within each validator the revenue is divided among delegators....

Reply:
ok